src/pages/About/Officers.vue

```diff
@@ -0,0 +1,226 @@
+        <h4 class="title is-4 has-text-centered">
+          The Board of Directors is the decision-making entity behind the OCF's
+          operations. The Board is appointed based on attendance at our
+          <a href="/about">weekly meetings</a>
+        </h4>
+        <div class="is-divider"></div>
+
+        <h2 class="title is-2 has-text-centered">General Managers</h2>
+        <h4 class="title is-4 has-text-centered">
+          General Managers handle the organizational activities of the OCF and
+          lead the Board of Directors meetings. They are elected by the Board of
+          Directors at the end of each semester.
+        </h4>
+        <OfficerCardSet
+          role-wanted="General Manager"
+          :cards="cards"
+          :double-set="false"
+        />
+        <br />
+        <div class="is-divider"></div>
+
+        <h2 class="title is-2 has-text-centered">Site Managers</h2>
+        <h4 class="title is-4 has-text-centered">
+          Site Managers lie behind the technical work involved in maintaining
+          the OCF lab and other services. They are elected by the Board of
+          Directors at the end of each semester.
+        </h4>
+        <OfficerCardSet
+          role-wanted="Site Manager"
+          :cards="cards"
+          :double-set="false"
+        />
+        <br />
+        <div class="is-divider"></div>
+        <h2 class="title is-2 has-text-centered">Committee Heads</h2>
+        <h4 class="title is-4 has-text-centered">
+          Committee heads guide the work of the various committees that function
+          within the OCF.
+        </h4>
+
+        <div class="columns is-centered">
+          <div class="column">
+            <h3 class="title is-3 has-text-centered">Internal Heads</h3>
+            <OfficerCardSet
+              role-wanted="Internal Head"
+              :cards="cards"
+              :double-set="true"
+            />
+          </div>
+          <div class="column">
+            <h3 class="title is-3 has-text-centered">
+              Industry and Alumni Relations Heads
+            </h3>
+            <OfficerCardSet
+              role-wanted="Industry and Alumni Relations Head"
+              :cards="cards"
+              :double-set="true"
+            />
+          </div>
+        </div>
+        <div class="columns is-centered">
+          <div class="column">
+            <h3 class="title is-3 has-text-centered">Finance Heads</h3>
+            <OfficerCardSet
+              role-wanted="Finance Head"
+              :cards="cards"
+              :double-set="true"
+            />
+          </div>
+          <div class="column">
+            <h3 class="title is-3 has-text-centered">Communications Heads</h3>
+            <OfficerCardSet
+              role-wanted="Communications Head"
+              :cards="cards"
+              :double-set="true"
+            />
+          </div>
+        </div>
+        <h3 class="title is-3 has-text-centered">DeCal Heads</h3>
```

Comment:
Even though there is only one column of cards here, it might make sense to wrap this part in with the columns as well, just to be consistent so that these are all wrapped in column divs and on the same level in the structure.